ql/src/test/results/clientnegative/materialized_view_authorization_rebuild_no_grant.q.out

```diff
@@ -33,4 +33,4 @@ POSTHOOK: type: CREATE_MATERIALIZED_VIEW
 POSTHOOK: Input: default@amvrng_table
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@amvrng_mat_view
-FAILED: HiveAccessControlException Permission denied: Principal [name=user1, type=USER] does not have following privileges for operation QUERY [[INSERT, DELETE] on Object [type=TABLE_OR_VIEW, name=default.amvrng_mat_view, action=INSERT_OVERWRITE]]
+FAILED: HiveAccessControlException Permission denied: Principal [name=user1, type=USER] does not have following privileges for operation ALTER_MATERIALIZED_VIEW_REBUILD [[OBJECT OWNERSHIP] on Object [type=TABLE_OR_VIEW, name=default.amvrng_mat_view, action=INSERT_OVERWRITE]]
```

Comment:
It seems we need object ownership for rebuild now (this is probably because of the changes in Operation2Privilege.java). We should mimic HiveOperationType.QUERY privileges instead of other ALTER statements (we probably need SELECT, INSERT, and DELETE on the MV).

Reply:
fixed